### PR TITLE
Fix ts types

### PR DESCRIPTION
**Pull request by ghost**
Also backport the changes that were only targeting dev/8.2.

### CODE REVIEW

**Review by KillianG — APPROVED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -21,6 +21,18 @@ interface LogDictionary {
 }
 
 declare module 'werelogs' {
+
+    class EndLogger {
```
Comment (francoisferrand) [inline index.d.ts:25]: should it not be exported as well?

**Review by francoisferrand — APPROVED** [review]

File: index.d.ts (hunk 1 above)
Comment (ghost) [reply index.d.ts:25]: We are not using it directly, it's returned by the `end()` method from the request logger instances, so we do not need to export it.

**Pull request merged**